apps/fis/aat/base/kustomization.yaml

```
@@ -2,6 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../base
+  - ../../../am/identity/identity.yaml
 namespace: fis
 patchesStrategicMerge:
   - ../../identity/aat.yaml
+  - ../../../am/identity/identity.yaml
```

this doesn't look right you've added an identity base and a patch with the same name?

are you meaning to do this:
```suggestion
  - ../../../am/identity/aat.yaml
```